From 10c93f58be89b011e063f3a4a129ec13339cbfc0 Mon Sep 17 00:00:00 2001
From: Ecco Park <eccopark@broadcom.com>
Date: Tue, 5 Nov 2013 07:41:18 +0900
Subject: [PATCH] net: wireless: bcmdhd: reduced the wakelock time of RX packet

	Reduce time : 1secs -> 500ms
	Problem : [Issue 11512235] 34% battery eaten overnight
		  [Issue 11374623] OS + Wifi used 41% of battery,
		  wlan_rx_wake
Signed-off-by: Ecco Park <eccopark@broadcom.com>

Signed-off-by: Francisco Franco <franciscofranco.1990@gmail.com>
---
 drivers/net/wireless/bcmdhd/dhd.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/bcmdhd/dhd.h b/drivers/net/wireless/bcmdhd/dhd.h
index 9ff39ce..3775ed0 100644
--- a/drivers/net/wireless/bcmdhd/dhd.h
+++ b/drivers/net/wireless/bcmdhd/dhd.h
@@ -482,7 +482,7 @@ inline static void MUTEX_UNLOCK_SOFTAP_SET(dhd_pub_t * dhdp)
 #define DHD_EVENT_TIMEOUT_MS	1500
 
 #if defined(CUSTOMER_HW4) && defined(PNO_SUPPORT)
-#define DHD_PNO_TIMEOUT_MS	10000
+#define DHD_PNO_TIMEOUT_MS	5000
 #endif
 
 /* interface operations (register, remove) should be atomic, use this lock to prevent race
-- 
1.8.4.2

